packages/breadcrumbs/src/styled/StyledChevronIcon.tsx

```diff
@@ -28,7 +28,8 @@ const ValidChevronIcon: React.FC<HTMLAttributes<SVGElement> & ThemeProps<Default
  */
 export const StyledChevronIcon = styled(ValidChevronIcon).attrs({
   role: 'presentation',
-  'aria-hidden': 'true'
+  'aria-hidden': 'true',
+  color: PALETTE.grey[600]
```

Comment:
To allow this to be themed dynamically, could we use `props.theme` within the styled component?

Something like:

```
import { getColor } from '@zendeskgarden/react-theming';

styled(...).attrs(...)`
  color: ${props => getColor('neutralHue', 600, props.theme)};
`
```

Reply:
Thanks for that tip. I've updated it!